addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java

```diff
@@ -68,6 +68,10 @@ public int getPluginId() {
 		return PLUGIN_ID;
 	}
 
+    public String getHelpLink() {
+        return "https://www.zaproxy.org/docs/desktop/addons/imagelocationscanner/#id-" + PLUGIN_ID;
+    }
+
```

Comment:
Indent to match surrounding code?

Reply:
I was actually wanting to remove the exception for this class and format it with spotless, any objection?